queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
Sure, updated in 6b150e2.

The way I've been thinking about where properties go for the other sources is more like a base set of properties that are common across all layers, and then the differences that get "mixed in" are what I've been putting in the layer specific namespace. The wof source is `places` layer only, so from this perspective it's somewhat arbitrary where the properties go. Anyway, happy to move them wherever they make sense and is consistent.